app/views/admin/announcements/_announcement.html.slim

```diff
@@ -1,4 +1,4 @@
-div.announcement class=time_period_class(announcement)
+= div_for(announcement, class: ['announcement', time_period_class(announcement)])
```

Comment:
If you specify the class, it will not add and just use the class you have specified.